applications/web3js-plugin-for-polkadot.md

```diff
@@ -0,0 +1,165 @@
+# Advancing Web3.js Plugin for Polkadot
+
+Propose to work more on the npm package: [`web3-plugin-polkadot`](https://www.npmjs.com/package/@conx3/web3-plugin-polkadot) and its website www.web3polka.com
+
+- **Team Name:** ConX3
+- **Payment Address:** 11xK5C1XUMsSUSAmguBoHTQLD8EguJCAxaTm1MnjDJ78Vdt (USDC or better in DOT)
```

Comment:
Just fyi, we don't offer payment in DOT.

Reply:
Ok 👍,
Thanks for the clarification,